src/app/shared/components/funding-progress/funding-progress.component.html

```diff
@@ -0,0 +1,25 @@
+<div *ngIf="fundsRaised < fundsRequired">
+  <span class="mt-6 font-bold text-gray-800 text-sm">{{ fundsRaised | currencyDefault }}</span>
+  <span class="mx-1 text-sm">of</span>
+  <span class="mt-6 font-bold text-gray-800 text-sm">{{ fundsRequired | currencyDefault }}</span>
+  <div class="relative pt-1 mt-2">
+    <div class="overflow-hidden h-2 mb-2 text-xs flex rounded bg-gray-200">
+      <div [style]="{ width: raisedPercentage() + '%' }"
```

Comment:
fyi, I've recently found out that you can write it as:
```
      <div [style.width.%]="raisedPercentage()"
```
... so you can try it. I'm not sure whether it works. Just something I found in some tweet.

Also, not so important now, but it can be in the future; when injecting data in templates, prefer reading from variables rather than executing functions. The problem is, when anything happens in one component, the entire app needs to be checked for changes, and when it happens, it checks every template whether it has changed. In this scenario, `raisedPercentage()` would be called on every check cycle and it might be a performance issue. In the default change detection, it happens all the time, but as we use OnPush change detection everywhere, it happens less frequently. It's good to know, but you won't notice it on basic screens.